backend-es/src/Main.purs

```diff
@@ -51,6 +51,7 @@ type BuildArgs =
   , foreignDir :: Maybe FilePath
   , directivesFile :: Maybe FilePath
   , intTags :: Boolean
+  , bareEnumSumTags :: Boolean
```

Comment:
I wonder if this is something that even needs to be configurable. Is there a reason why one wouldn't want this?

Reply:
I wouldn't want to configure this. I think this is an optimization that should be done by default.

Reply:
I totally agree. Should I remove the option?

Reply:
I think the option can be removed.

Reply:
Done.